guides/source/active_support_core_extensions.md

```diff
@@ -3399,18 +3399,19 @@ NOTE: Defined in `active_support/core_ext/date_and_time/calculations.rb`.
 [DateAndTime::Calculations#months_ago]: https://api.rubyonrails.org/classes/DateAndTime/Calculations.html#method-i-months_ago
 [DateAndTime::Calculations#months_since]: https://api.rubyonrails.org/classes/DateAndTime/Calculations.html#method-i-months_since
 
-##### `weeks_ago`
+##### `weeks_ago` and `weeks_since`
```

Comment:
Just following the previous sections, let's use a comma here:

```suggestion
##### `weeks_ago`, `weeks_since`
```

Reply:
Thank you for reviewing. Fixed as you suggested.